### PR TITLE
More Padding Stuffs

### PR DESCRIPTION
**Pull request by wylie**
Cleaning up the grid a little more.

### CODE REVIEW

**Comment by wylie** [comment]
I am going to leave this overnight so @gjjones can take a gander before it gets merged into develop. I think this is it though. Everything looks fine on my end and in all the browsers I've checked.

:balloon:

File: less/_grid.less [hunk 1]
```diff
@@ -76,19 +76,17 @@
 
 // other CSS for the container of the new columns
 .grid {
-	@grid-sides: 15px;
+
 	.clearfix;
 
-	padding: 0 25px 40px 25px;
+	padding-top: 0;
```
Comment (gjjones) [inline less/_grid.less]: Shouldn't think be 0 by default?

File: less/_variables.less [hunk 2]
```diff
@@ -214,6 +214,11 @@
 // FORMS
 
 
+// GRID
+@grid-sides: 20px;
+@side-gutter: 25px;
+@foo: 100px;
```
Comment (gjjones) [inline less/_variables.less]: Was it intended to leave this in the PR?

File: less/_mixins.less [hunk 3]
```diff
@@ -96,13 +93,20 @@
...
 	overflow: hidden;
 }
-
 .off-page {
 	display: inline-block;
 	font-size: 0;
 	text-indent: -99999px
 }
 
+//----- VISIBLE THINGS ---------------------------------//
```
Comment (gjjones) [inline less/_mixins.less:102]: Should I assume this isn't related to the PR?

File: less/_grid.less (hunk 1 above)
Comment (wylie) [inline less/_grid.less]: Good point. It shouldn't have any styles anywhere else. Will check into it.

File: less/_mixins.less (hunk 3 above)
Comment (wylie) [inline less/_mixins.less:102]: Correct, @amyh added that. Not sure why it's showing that I did.

File: less/_variables.less (hunk 2 above)
Comment (wylie) [inline less/_variables.less]: oops, will remove that when I get in.

Comment (wylie) [inline less/_variables.less]: Removed!

File: less/_grid.less (hunk 1 above)
Comment (wylie) [inline less/_grid.less]: Wasn't needed, removed!

**Comment by wylie** [comment]
Seems ready to merge, also seems ready for prime time.

**Comment by gjjones** [comment]
We rock!

**Pull request merged**